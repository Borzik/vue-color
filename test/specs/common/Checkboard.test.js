import { shallow } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'

import Checkboard from '../../../src/components/common/Checkboard'

describe('Checkboard.vue', () => {
  let wrapper
  let renderer

  beforeEach(() => {
    wrapper = shallow(Checkboard)
    renderer = createRenderer()
  })

  test('renders correctly', () => {
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
