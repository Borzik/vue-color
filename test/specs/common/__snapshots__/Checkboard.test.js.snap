// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkboard.vue renders correctly 1`] = `<div class="vc-checkerboard" style="background-image:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAAAM0lEQVQ4jWN89uzZfwY8QFJSEp80AxNeWSLAqAGDwQDG////400Hz58/p60LRg0YDAYAAJKtCsxkMGMsAAAAAElFTkSuQmCC);"></div>`;
